### PR TITLE
Add TS typings

### PR DESCRIPTION
**Pull request by sebald**
Hey, noticed that there are no typings yet. So I made some :)

### CODE REVIEW

File: index.test-d.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import * as React from 'react';
 import InkSelect, { Item } from '.';
 
-const plain = () => <InkSelect/>;
-const withProps = () => <InkSelect focus={false} limit={10} />;
+const plain = () => (<InkSelect/>);
```
Comment (vadimdemedes) [inline index.test-d.tsx]: You don't need to wrap JSX if it's only one line ;)

File: index.test-d.tsx [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+const withProps = () => (<InkSelect focus={false} limit={10} />);
 
-const items = () => <InkSelect items={[{ label: 'label 1', value: 'label 1'}, { label: 'label 2', value: 'label 2'}]}/>
-const itemsWithKey = () => <InkSelect items={[{ label: 'label 1', value: 'label 1', key: 1 }, { label: 'label 2', value: 'label 2', key: 2 }]}/>
+const items = () => (<InkSelect items={[{ label: 'label 1', value: 'label 1'}, { label: 'label 2', value: 'label 2'}]}/>);
```
Comment (vadimdemedes) [inline index.test-d.tsx]: Could you extract array of items here to `items` variable and reuse it instead?

**Comment by sebald** [comment]
@vadimdemedes Update. I ran prettier to format the file :D

**Comment by vadimdemedes** [comment]
Could you configure prettier to follow the same rules as the rest of the codebase? For example, single quotes.

**Comment by sebald** [comment]
@vadimdemedes sorry, didn't thought about this. Hope I got it right this time :)

Use the following config:

```json
{
  "arrowParens": "avoid",
  "bracketSpacing": false,
  "htmlWhitespaceSensitivity": "css",
  "insertPragma": false,
  "jsxBracketSameLine": false,
  "jsxSingleQuote": true,
  "parser": "typescript",
  "printWidth": 80,
  "proseWrap": "preserve",
  "requirePragma": false,
  "semi": true,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "none",
  "useTabs": true
}
```

[Playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLAtgBwgJxgAgCp8BDAZ3wCU4SwCAzHCdfAHRBxrvYG5WoM2PPgCSUANYBlOABs4dADT5go+OiViAJqjAkYuAApNMZDWqMQT+AL75GzNiAB0vfv0hQyBVGrJJVcOgA2gC6+AC8+EH8+Mr4MiQARrL+7AnJMvgAjOxKAG4kMgCucKkg6bLZ7DYKMXEVMmUN+ABMufgFxaWOzW0gNvwhfFAeXvjicACecJoivv5zgaERUXUqDU1JlTkg+YUlmxlVu+NT-lk1a-FbjT03re2dB3dHfUoTk-4tA1BDbiPQMaYBKoKArAAUAEoIgA+fAAHjEUlk8gIAHoYcNRgQAO4+AAWFiskShsIRSOkcjodggYCKZHCwHohTIcFsMgwPkZWQADLYMcN3ICCKyqTAAOoExboCgk6HhOGIiSU1H4HyBBnAdUy-mYoWeEUouiSmD4gDSUxm0tl+FJCvJyqN3l8jI+Vt8usFAIN+GgKupJO1CzU8rhowgcicMggAHNwdrIVjhfg8aaABIkKCaOQ4CGhh3IsW+qD+mCMv1Oz3-bH4ADC9P06C0qHEelw-motBgTgAYrWldpdPocESyHCSSpUGRSzMbPnwXV4do8jCtdOnbOAPyOQDI5NUyuxrPC0cuYfxE-qxvWvMxpbXmNgoAgYB2uN2+4jzMYxxDJ+uxTMSjNNY86LqedSxGuM6aPg27sHu-QHiAtjAA01iLieqArueSY+hAeRwDgOCoJocD3lg0DPjadpwguYIFqWEFqlmOhtjg5GPs+jLXo2zatsO6H0bE2ocZRsDcQ2t5qKJT6wIJsQYjh-C7CAlgwKggLIKAJBERAOIGDpCBkMgIAkHkEAkSpiQ4LQEwwJImC0KCMbIMyMisgoID4jA6AyOK+LqmQjlgHA0ieD4WE+JMJlgGQxmeaCrJ4EYJAxugJCuSycCeQAVmQAAeABCNlgHZkgkOgcAADKgnAmXudlIB5flkjOXIACKRQQPAyAwDgJSeY5OBJSZMCTJgcBkGAxGYDAKmYMRsCSpoprIAAHDyg1MKy4o2ZgJkLZNhEESpnAAI5FKgnApWlGVIG5HkgKy6CoL1-WNWQbVwJ13V1UgfUDSAMBJMtq1IC0nl9SQqAclAMbkelJlQJRKn0nAAAqSTGf973WNYQA)

**Comment by vadimdemedes** [comment]
LGTM, thanks!

**Pull request merged**